cilium/cmd/bpf_metrics_list_test.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2020 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !privileged_tests
+
+package cmd
+
+import (
+	"encoding/json"
+	"sort"
+	"strings"
+
+	"github.com/cilium/cilium/pkg/checker"
+	"github.com/cilium/cilium/pkg/maps/metricsmap"
+	monitorAPI "github.com/cilium/cilium/pkg/monitor/api"
+	"github.com/cilium/cilium/pkg/testutils/mockmaps"
+
+	. "gopkg.in/check.v1"
+)
+
+type BPFMetricsmapuite struct{}
+
+var _ = Suite(&BPFMetricsmapuite{})
+
+func (s *BPFMetricsmapuite) TestDumpMetrics(c *C) {
```

Comment:
This is still left over: https://github.com/cilium/cilium/pull/13731#discussion_r521580547

Reply:
:man_facepalming: :man_facepalming: 
time for larger terminal fonts I guess (sorry for the noise, just updated the PR)